wrapper/src/main/java/software/amazon/jdbc/profile/ConfigurationProfilePresetCodes.java

```diff
@@ -52,6 +54,14 @@ public class ConfigurationProfilePresetCodes {
   public static final String I0 = "I0"; // Normal
   public static final String I1 = "I1"; // Aggressive
 
+  public static final String SF_D0 = "SF_D0"; // Normal
```

Comment:
NIT:
what about SD0, or S_D0?
however, it would be nice to add an extra comment, to indicate whether S_D0 and D0 are related just with the addition of Spring Framework

Reply:
Line 31 the same file?